website/pages/docs/secrets/venafi/index.mdx

```diff
@@ -0,0 +1,270 @@
+---
+layout: docs
+page_title: Venafi - Secrets Engines
+sidebar_title: Venafi (Certificates)
+description: The Venafi integrated secrets engine for Vault.
+---
+
+# Venafi Secrets Engine for HashiCorp Vault
+
+The Venafi Machine Identity Secrets Engine provides applications with the
+ability to dynamically generate SSL/TLS certificates that serve as machine
+identities. Using
+[Venafi Trust Protection Platform](https://www.venafi.com/platform/trust-protection-platform)
+or [Venafi Cloud](https://www.venafi.com/venaficloud) assures compliance
+with enterprise policy and consistency with industry standard trust protection.
+Designed for high performance with the same interface as the built-in PKI
+secrets engine, services can get certificates without manually generating a
+private key and CSR, submitting to a certificate authority, and waiting for a
+verification and signing process to complete. Venafi's certificate authority
+integrations and policy controls, combined with Vault's built-in authentication
+and authorization mechanisms, provide the verification functionality.
+
+Like the built-in PKI secrets engine, short-lived certificates for ephemeral
+workloads are the primary focus of the Venafi secrets engine. As such,
+revocation is not currently supported.
+
+The Venafi secrets engine makes use of HashiCorp Vault's
+[plugin system](https://www.vaultproject.io/docs/internals/plugins.html)
+and Venafi's [VCert Client SDK](https://github.com/Venafi/vcert).  If you're a
+developer and you are interested in contributing to this secrets engine, visit
```

Comment:
It would be good to expand on this a little to further direct questions and/or issue reports to that plugin repo too.

Reply:
Done